pkg/kopia/command/repository_connect.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package command
+
+import (
+	"time"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/pkg/errors"
+
+	"github.com/kanisterio/kanister/pkg/kopia"
+)
+
+type RepositoryConnectCommandArgs struct {
+	*CommandArgs
+	Prof                  kopia.Profile
+	ArtifactPrefix        string
```

Comment:
As suggested above, I think we should use `PathPrefix` or `RepoPathPrefix` instead of `ArtifactPrefix`